app/views/static_page/help.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= provide:title, "Help" %>
+
+<h1>Help</h1>
+<p>
+ Get help on the Ruby on Rails Tutorial at the
+ <%= link_to "Rails Tutorial Help page", "https://www.railstutorial.org/help" %>
```

Comment:
with the link, you should use gem config to manage them. 
https://github.com/rubyconfig/config